src/pages/Governance/Types.ts

```diff
@@ -4,10 +4,7 @@ export type Proposal = {
   proposer: string;
   creation_time_secs: string;
   execution_content: {
```

Comment:
If we don't need it anymore, just remove this field?

Reply:
It is not deprecated on the move module https://github.com/aptos-labs/aptos-core/pull/3140/files#diff-5b53920d43083fa179b5920517df200f09135ca238f81d8971a45efd7ee5936eR60 so thought we might want/need to have it around